packages/api/tests/endpoints/providers.js

```diff
@@ -193,28 +197,50 @@ test('CUMULUS-912 DELETE with pathParameters and with an unauthorized user retur
   });
 });
 
-// TODO(aimee): Add a provider to ES. List uses ES and we don't have any providers in ES.
-test('default returns list of providers', (t) => {
-  const listEvent = {
-    httpMethod: 'list',
-    headers: authHeaders
+test('POST with invalid authorization scheme returns an invalid token response', (t) => {
+  const request = {
+    httpMethod: 'POST',
+    headers: {
+      Authorization: 'InvalidBearerScheme ThisIsAnInvalidAuthorizationToken'
+    }
   };
 
-  return testEndpoint(providerEndpoint, listEvent, (response) => {
-    const { results } = JSON.parse(response.body);
-    t.is(results.length, 0);
+  return testEndpoint(providerEndpoint, request, (response) => {
+    assertions.isInvalidTokenResponse(t, response);
   });
 });
 
-test('GET returns an existing provider', (t) => {
-  const getEvent = {
-    httpMethod: 'GET',
-    pathParameters: { id: testProvider.id },
+test('POST with non-expiring operator credentials returns an invalid token response', async (t) => {
+  const fakeUser = fakeUserFactory();
+  const authToken = (await userModel.create(fakeUser)).password;
+  await userModel.update({ userName: fakeUser.userName }, {}, ['expires']);
+
+  const request = {
+    httpMethod: 'POST',
+    headers: {
+      Authorization: `Bearer ${authToken}`
+    }
+  };
+
+  return testEndpoint(providerEndpoint, request, (response) => {
+    assertions.isInvalidTokenResponse(t, response);
+  });
+});
+
+test('default returns list of providers', async (t) => {
+  const newProviderId = randomString();
+  const newProvider = Object.assign({}, testProvider, { id: newProviderId });
+
+  await indexProvider(esClient, newProvider, esIndex);
+
+  const listEvent = {
+    httpMethod: 'list',
```

Comment:
"list" is not an http method.  This should be "GET".

Reply:
Indeed. I think I actually need to omit `httpMethod` so it triggers the `list` handler for the endpoint

Reply:
The request object we send into the endpoint should be a valid request object, so the method should be GET. The fact that it's a default is an implementation detail. I should be able to change that "if" statement, put the "list" option at the top, and the test should still pass.